server/controllers/bookingControllers.js

```diff
@@ -37,86 +35,39 @@ class bookingsController {
   { 
       // check if trip is activated*
     if (findtripid.rows[0].status == 'CANCELED') {
-      return response.response(
-        res,
-        406,'error',
-  
-        'TRIP HAS BEEN CANCELED!!! PLEASE TRY ANOTHER DIFFERENT TRIP',
-        true
-      );
-    }
+      return response.response( res,406,'error','TRIP HAS BEEN CANCELED!!! PLEASE TRY ANOTHER DIFFERENT TRIP',true);}
+ }
 
+ const{bus_license_number,origin,destination,fare,time,id,trip_date,seating_capacity}=findtripid.rows[0]
 
- }
- 
-  const buslicence = findtripid.rows[0].bus_license_number;
-  const origin  = findtripid.rows[0].origin;
-  const destination  = findtripid.rows[0].destination;
-  const fare = findtripid.rows[0].fare;
-  const triptime = findtripid.rows[0].time;
-  const tripidnumber = findtripid.rows[0].id;
-  const tripdate = findtripid.rows[0].trip_date;
-  const seating_capacity = findtripid.rows[0].seating_capacity;
-
-
-  // // ###check users  req.user.id
+ // // ###check users  req.user.id
   let userCheck = await client.query('SELECT * FROM users WHERE id=$1 ',[
     req.user.id,
   ]);
   const userid = req.user.id;
```

Comment:
Why not define this above userCheck variable and then use it

Reply:
ok